### PR TITLE
add faq

### PR DESCRIPTION
**Pull request by mirnawong1**
per slack thread flag by erin and miguel, this pr adds faq steps to clarify how to migrate from one git provider to another in dbt cloud. it also links the git faq on particular git pages.

### CODE REVIEW

**Comment by mirnawong1** [comment]
Still working on this pr and will finalize Fri

**Comment by mirnawong1** [comment]
going to tackle this monday

**Comment by mirnawong1** [comment]
hey @b-per, i've created this pr per your instructions [here](https://dbt-labs.slack.com/archives/C017GDLAF7D/p1695129929253769). i had one comment for step 2 as i wasn't sure i understood - could you give it a 👀  please?

File: website/docs/faqs/Git/git-migration.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+tags: [Git]
+---
+
+To migrate from one git provider to another, refer to the following steps to avoid minimal disruption:
+
+1. Outside of dbt Cloud, you'll need to import your existing repo into your new provider. 
+   
+   As an example, if you're migrating from GitHub to Azure DevOps, you'll need to import your existing repo (GitHub) into your new git provider (Azure DevOps). For detailed steps on how to do this, refer to your git provider's documentation (Such as [GitHub](https://docs.github.com/en/migrations/importing-source-code/using-github-importer/importing-a-repository-with-github-importer), [GitLab](https://docs.gitlab.com/ee/user/project/import/repo_by_url.html), [Azure DevOps](https://learn.microsoft.com/en-us/azure/devops/repos/git/import-git-repository?view=azure-devops)) 
+   
+2. Go back to dbt Cloud and set up your [integration for the new git provider](/docs/cloud/git/connect-github), if needed.
```
Comment (mirnawong1) [inline website/docs/faqs/Git/git-migration.md:16]: @b-per , not sure if step 2 is the same as step 4 - is step 2 needed IF they don't have an integration set up in dbt cloud already? or are we asking the users to integrate with the new git provider twice?

Comment (b-per) [reply website/docs/faqs/Git/git-migration.md:16]: Steps 2 and 4 are definitely different, so it looks good to me.

Step 2 is about setting up the integration (at most required once per account) whereas step 4 is connection the dbt Cloud project to the repository (required for each dbt Cloud project, so potentially more than once)

File: website/docs/faqs/Git/git-migration.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+title: "How to migrate git providers"
+sidebar_label: "Git Migration"
+id: "git-migration"
+description: "Learn how to migrate git providers in dbt Cloud with minimal disruption."
+tags: [Git]
+---
+
+To migrate from one git provider to another, refer to the following steps to avoid minimal disruption:
+
+1. Outside of dbt Cloud, you'll need to import your existing repo into your new provider.
```
Comment (b-per) [inline website/docs/faqs/Git/git-migration.md]: Do we use "repo" in other parts of the docs or do we usuall refer to "repository". (repo appears a few times here)

Comment (mirnawong1) [reply website/docs/faqs/Git/git-migration.md]: we do use repo all over the docs, you can shorten words if you introduce the main word at the beginning. but I'll update to repository for consistency on this page.

**Comment by mirnawong1** [comment]
thank you again @b-per ! i've fixed this up for consistency - when you have a chance, can you give it another 👀 ? thanks so much!

File: website/static/img/docs/dbt-cloud/on-premises/disconnect-repo.gif
Comment (b-per) [inline website/static/img/docs/dbt-cloud/on-premises/disconnect-repo.gif:1]: This file is not necessary is it?

**Review by b-per — APPROVED** [review]
All good with the exception that I think the GIF has been uploaded twice

File: website/static/img/docs/dbt-cloud/on-premises/disconnect-repo.gif
Comment (mirnawong1) [reply website/static/img/docs/dbt-cloud/on-premises/disconnect-repo.gif:1]: omg i wondered where the file went! i thought it dragged it and then it disappeared so i did it again. great catch, thank you! removed

**Comment by mirnawong1** [comment]
merging this now, thank you so much @b-per ✨

**Pull request merged**